js/src/collapse.js

```diff
@@ -314,6 +325,31 @@ const Collapse = (($) => {
       }
     }
 
+    _getShowClass() {
+      const tabClass = this._element.classList
+
+      // Detect flex for inline style
+      let useFlex = $(this._element).css('display').indexOf('flex') !== -1
```

Comment:
I looked into it a bit more and it seems Babel doesn't do this. Just verify one more time and if it doesn't just use `indexOf`. It's not worth using a polyfill for this.